internal/services/hdinsight/hdinsight_hadoop_cluster_resource.go

```diff
@@ -95,6 +95,8 @@ func resourceHDInsightHadoopCluster() *pluginsdk.Resource {
 				},
 			},
 
+			"disk_encryption_properties": SchemaHDInsightsDiskEncryptionProperties(),
```

Comment:
i think properties is redundant?
```suggestion
			"disk_encryption": SchemaHDInsightsDiskEncryptionProperties(),
```